### DISCUSSION TITLE
Karavan UI basic auth

### DISCUSSION THREAD

**Discussion by t33muki** [apache/camel-karavan#1397] [Q&A]
### Ask a question

How to enable basic auth on Karavan 4.7.1? 

### Variant

Web Application

### Container Management (if applicable)

Kubernetes

### Operating System (if applicable)

Linux

### Version

None

**Comment by mgubaidullin** [comment]
The latest releases version of Karavan is 4.7.0

Karavan does not provide basic authentication
